CHANGELOG.md

```diff
@@ -7,7 +7,8 @@ All notable changes to **NCronJob** will be documented in this file. The project
 ## [Unreleased]
 
 ### Added
-- Ability to add a timezone for a "minimal job"
+- Ability to add a timezone for a "minimal job".
+- Run jobs automatically when a job either succeeded or failed allowing to model a job pipeline. By [@linkdotnet](https://github.com/linkdotet).
```

Comment:
Your Github link is broken ;-)
Missing `n` in `net`

Reply:
Thanks and done!